ISISComputingGroup/EPICS-kepco
# Change units of voltage to V rather than A

### Description of work

Changed units of voltage from A to V in the Kepco IOC

### To test

https://github.com/ISISComputingGroup/IBEX/issues/1939

Stop IBEX server, re-make the Kepco support module, then restart IBEX server to see changes.

### Acceptance criteria

- [ ] Units of Voltage have been changed from A to V in the Kepco IOC

---

#### Code Review


### Functional Tests


### Final steps

- [x] Reviewer has updated the submodule in the main EPICS repo, **master branch**? See **Reviewing work for the subModules of EPICS** in the [Git workflow](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Git-workflow) page for details.
